sriovnet.go

```diff
@@ -520,3 +520,32 @@ func GetPciFromNetDevice(name string) (string, error) {
 	}
 	return base, nil
 }
+
+// GetPKeyFromPci returns the index0 PKey for the IB PCI device
+func GetPKeyFromPci(pciAddress string) (string, error) {
```

Comment:
this works for both PF and VFs ?

Reply:
there are several indexes for pkey membership, index0 is the default membership.

should we have two functions ?

`func GetPkeyByIndexFromPci(pciAddress string) (string, error)`

and

`func GetDefaultlPkeyFromPci(pciAddress string) (string, error`

Reply:
This works for both PFs and VFs as the mechanism around how PKeys are stored is the same for both.
I'm not opposed having two functions to retrieve different PKeys. Let me address that in the PR

Reply:
Split into two new functions